script/npm_link_foreman_js.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ -z "${FOREMAN_JS_LOCATION}" ]]; then # FOREMAN_JS_LOCATION is empty
+  FOREMAN_JS_LOCATION="../foreman-js"
```

Comment:
wouldn't' ../ refer to the script path? imho you need to take into account the following

* add "set -x" # so it exits the script if something returned an error
* print the actual paths used, e.g. FOREMAN_JS_LOCATION is not defined, using <x> instead.
* check that <x> actually exists

Also, how would a user undo this action? a simple npm install will fix it?

Reply:
@ohadlevy see the doc about using `foreman-vendor`:
https://github.com/theforeman/foreman/pull/6605/files#diff-c681f7a6d59f3d900f78ad1aea482096


> add "set -x" # so it exits the script if something returned an error
print the actual paths used, e.g. FOREMAN_JS_LOCATION is not defined, using instead.
check that actually exists

+1

> Also, how would a user undo this action? a simple npm install will fix it?

 Yes

Reply:
Thanks @ohadlevy, updated